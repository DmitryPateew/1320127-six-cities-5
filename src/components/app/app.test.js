import {expect, it} from "@jest/globals";
import Enzyme, {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter: new Adapter()});

import React from "react";
import App from "./app";

it(`Should app render correctly`, () => {
  const app = shallow(<App/>);
  expect(shallowToJson(app)).toMatchSnapshot();
});
