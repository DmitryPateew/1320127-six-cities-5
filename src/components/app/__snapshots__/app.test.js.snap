// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should app render correctly 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(MainPage)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      exact={true}
      path="/login"
    >
      <Connect(LoginPage) />
    </Route>
    <Connect(PrivateRoute)
      exact={true}
      path="/favorites"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/offer/:id"
    >
      <Connect(withRouter(OfferPage)) />
    </Route>
  </Switch>
</Router>
`;
